Scripts/update_discovery_resources.pl

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/perl -w
```

Comment:
Why perl?

Reply:
It's the scripting language I'm most comfortable with, and it ships with macOS. My Python is passable, and my Ruby is pretty bad. If you prefer some other language over Perl, let me know.